pages/core-api/safe-installation-guide.mdx

```diff
@@ -0,0 +1,250 @@
+    ```json
+    "<your_chain_id>": "canonical"
+    ```
+
+3. Open a pull request. Your pull request should follow this [example pull request](https://github.com/safe-global/safe-deployments/pull/679).
+
+</Steps>
+
+## Safe\{Core\} Infrastructure deployment
+
+Once you have deployed the Safe\{Core\} contracts, you must deploy the off-chain components of the Safe\{Core\} Stack, including backend and frontend services. 
+
+You have three options for deploying the Safe stack, depending on your needs.
+
+### Option 1: Safe Core Contributor Platform-as-a-Service
+
+We roll out new networks quarterly, depending on our internal capacity, and only for medium to large chains based on a strict scoring framework. 
+
+If you want to submit your chain for assessment in the next quarter, please fill out this [form](https://noteforms.com/forms/request-safe-ui-and-infra-support-4weugt).
+
+#### Hard requirements
+
+- EVM-compatible chain.
+- At least two dedicated RPC node providers, with preference for at least one Tier 1 provider of Infura, Alchemy, Quicknode. You can find more on RPC nodes [here](https://docs.safe.global/core-api/api-safe-transaction-service/rpc-requirements).
+
+### Option 2: Third-party integrators
+
+We are happy to introduce you to our official integrators.
+
+List of official integrators:
+
+    - integrator A | website | telegram | twitter
+    - integrator B | website | telegram | twitter
+    - integrator C | website | telegram | twitter
+
+### Option 3: Self-hosting
+
+If you have engineering and Cloud resources, you can also deploy the different components of the Stack by following one of the procedures below. 
+
+You can find docker images for a majority of platforms. 
+
+#### Requirements
+
+**Hardware**
+
+These are the hardware requirements to run the different services required.
+
+
+| Network | Transaction Service | Client Gateway | Config Service | Database |
+| --- | --- | --- | --- | --- |
+| Small (new chain with low traffic) | CPU: 2 vCPU RAM: 8GiB | CPU: 2 vCPU RAM: 8 GiB | CPU: 2 vCPU RAM: 8 GiB | CPU: 1 cores/ 2 vCPU RAM: 8GiB |
+| Standard | CPU: 4 vCPU RAM: 16GiB | CPU: 2 vCPU RAM: 8 GiB | CPU: 2 vCPU RAM: 8 GiB | CPU: 2 cores/ 4 vCPU RAM: 16GiB |
+| Large (large chain with high-traffic) | CPU: 8 vCPU RAM: 32GiB | CPU: 2 vCPU RAM: 8 GiB | CPU: 2 vCPU RAM: 8 GiB | CPU: 8 cores/ 16 vCPU RAM: 64GiB |
+
+
+**RPC**
+
+For support, please refer to the [RPC requirements](https://docs.safe.global/core-api/api-safe-transaction-service/rpc-requirements#what-are-the-rpc-requirements). Safe (L1) requires tracing for indexing, while Safe (L2) supports events/logs.
+
+#### Docker-compose deployment
+
+Please see the [safe-infrastructure](https://github.com/safe-global/safe-infrastructure) repository for all required information.
+
+```bash
+git clone git@github.com:safe-global/safe-infrastructure.git
+cd safe-infrastructure
+```
+
+**Prerequisites**
+
+- Docker v20.10+
```

Comment:
Can we link to places to download these tools/software?

Reply:
Good idea, added the links.